docs/api/models.rst

```diff
@@ -13,16 +13,16 @@ Module containing base models and functions to operate on them.
     :exclude-members: __init__, __new__
     :members:
 
-.. autoclass:: DeployAccount
+.. autoclass:: DeployAccountV1
     :exclude-members: __init__, __new__
 
-.. autoclass:: Declare
+.. autoclass:: DeclareV1
     :exclude-members: __init__, __new__
 
 .. autoclass:: DeclareV2
     :exclude-members: __init__, __new__
 
-.. autoclass:: Invoke
+.. autoclass:: InvokeV1
```

Comment:
Other (new) models will be added in separate PR, right?

Reply:
Correct, it will be done with other documentation updates related to RPC v0.6.0.